docker-compose.yml

```diff
@@ -14,7 +14,21 @@ services:
     volumes:
       - "/srv/hommod:/data"
 
-  celery:
+  celery-beat:
```

Comment:
http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#starting-the-scheduler